codeworld-api/src/CodeWorld/Driver.hs

```diff
@@ -443,6 +444,11 @@ picToObj' pic =
             ptsJS <- pointsToArr pts
             setProps [("points", ptsJS), ("smooth", pToJSVal smooth)] obj
             retVal obj
+        SolidClosedCurve cs pts smooth -> do
+            obj <- init "SolidClosedCurve"
```

Comment:
Let's stick to the exported function name?  So `solidClosedCurve` rather than `SolidClosedCurve`.